trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -28,11 +28,27 @@ const getCompany = async (req, res) => {
   }
 };
 
-const getPartyUrn = async ({ companyRegNo }) => {
+const getPartyUrn = async ({ companyRegNo, companyName }) => {
```

Comment:
Why this function is being changed, only change I am expecting is at `APIM` end none at DTFS for a `GET` endpoint. Also why would we need to send `companyName` in argument?

Reply:
The company name is a required field for Salesforce account creation, so we pass it through here. this means we're not dependent on doing a later check by companies house number to pull the company name. We _could_ get it from a check MDM runs to the Dun and Bradstreet API, but that felt needless to me when we already have it in DTFS.

Reply:
Understood, will discuss other questions separately.